Makefile.standalone

```diff
@@ -0,0 +1,52 @@
+
+## path configuration (FHS-style)
+PREFIX ?= /usr
+BINDIR ?= $(PREFIX)/bin
+SBINDIR ?= $(PREFIX)/sbin
+
+## toolchain commands
+export GO ?= go
+export INSTALL ?= install
+export LN_S ?= ln -s
+export MKDIR_P ?= mkdir -p
+
+BUILD_TMP ?= $(CURDIR)/tmp
+BUILD_OUT ?= $(BUILD_TMP)/build
+export GOCACHE ?= $(BUILD_TMP)/go-cache
+export LDFLAGS=-s -w
+
+BUILD_GO_ARCH := $(shell $(GO) env GOARCH | sed -e 's~^386~i386~;')
+BUILD_GO_OS := $(shell $(GO) env GOOS)
+
+EXEC_NAME=dockerd-$(BUILD_GO_OS)-$(BUILD_GO_ARCH)
+EXEC_OUT=$(BUILD_OUT)/$(EXEC_NAME)
+
+# make sure it fails ...
+export GOPATH=/proc/gopath
+export GOPROXY=127.0.0.1:29999
+
+all: compile
+
+compile: $(EXEC_OUT)
+
+## generate go.mod in order to use simpler module build mode
+$(CURDIR)/go.mod:
+	$(GO) mod init
+
+$(CURDIR)/vendor/modules.txt:
+	touch $@
+
+$(EXEC_OUT): $(CURDIR)/go.mod $(CURDIR)/vendor/modules.txt
+	$(GO) build -ldflags "$(LDFLAGS)" -mod=vendor -o $@ $(CURDIR)/cmd/dockerd
+
+clean:
+	rm -Rf $(CURDIR)/vendor/modules.txt $(CURDIR)/go.mod $(CURDIR)/tmp
+
+install: compile
+	$(INSTALL) -D -m644 $(CURDIR)/contrib/udev/80-docker.rules        $(DESTDIR)/lib/udev/rules.d/80-docker.rules
```

Comment:
Should not be needed on most recent distros

Reply:
> Should not be needed on most recent distros

Define "most recent". Yes, udisks is was a ridicolous thing to begin with, but still gnome stuff (eg. gvfs) pulls it in.
And we certainly don't wanna have it touching the docker managed loop devs.

Reply:
> Define "most recent". 

https://github.com/docker/docker-ce-packaging/tree/master/rpm
https://github.com/docker/docker-ce-packaging/tree/master/deb

Reply:
> > Define "most recent".
> 
> https://github.com/docker/docker-ce-packaging/tree/master/rpm
> https://github.com/docker/docker-ce-packaging/tree/master/deb

This homebrewn stuff in no way fits into distro build infrastructure. That's exactly why I'm doing these patches.